okapi-core/src/main/java/org/folio/okapi/managers/HealthManager.java

```diff
@@ -0,0 +1,68 @@
+package org.folio.okapi.managers;
+
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.http.HttpServerOptions;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import java.util.List;
+import org.folio.okapi.service.Liveness;
+
+public class HealthManager {
+  private Vertx vertx;
+  private final int listenPort;
+  private List<Liveness> livenessChecks;
+
+  public HealthManager(int listenPort) {
+    this.listenPort = listenPort;
+  }
+
+  /**
+   * Initialize health manager.
+   * This should be called when the service should start
+   * serving readiness and liveness.
+   * @param vertx Vert.x handle
+   * @return future result
+   *
+   */
+  public Future<Void> init(Vertx vertx, List<Liveness> livenessChecks) {
+    this.vertx = vertx;
+    this.livenessChecks = livenessChecks;
+    if (listenPort == 0) {
+      return Future.succeededFuture();
+    }
```

Comment:
The if clause should be move to the first line to avoid storing livenessChecks if they are not needed.

Reply:
ok